scripts/core/exp-bindless-images.yml

```diff
@@ -176,6 +179,19 @@ members:
       desc: "[in] mipmap filter mode used for filtering between mipmap levels"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Describes unique sampler addressing mode per dimension"
+details:
+    - Specify these properties in $xSamplerCreate via $x_sampler_desc_t as part
+      of a `pNext` chain.
+class: $xBindlessImages
+name: $x_exp_sampler_addr_modes_t
+base: $x_base_properties_t
+members:
+    - type: $x_sampler_addressing_mode_t
+      name: addrModes[3]
```

Comment:
Unfortunatelly this won't work with the current generator. I think the simplest solution would be to just have three named fields for the addressing mode values.

Reply:
I've created https://github.com/oneapi-src/unified-runtime/issues/869 to add this support.

Reply:
Great, thanks! We'll keep an eye on this but will use the separate variables as a temporary solution.